### ISSUE TITLE
2.4.7 Focus Visible: WCAG 2.1 Understanding has unmarked WCAG 2.2 sentence

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1177]
Follow up from #1162:

The Understanding document for “Focus Visible” has an unmarked section that only applies to WCAG 2.2. It should either be removed or clearly marked as WCAG 2.2.

> This criterion does not specify what the form is, but Focus Visible (Enhanced) does define how visible the indicator should be. Passing Focus Visible (Enhanced) would pass this success criterion.

https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html
